scripts/globals/equipment.lua

```diff
@@ -124,24 +279,24 @@ xi.equipment.relicIDs =
 local artifactArmorRanges =
 {
 --      Min,   Max
-    { 12511, 12520 }, -- Original Head
-    { 13855, 13857 },
-    { 13868, 13869 },
-    { 12638, 12650 }, -- Original Body
-    { 13781, 13782 },
-    { 13961, 13975 }, -- Original Hand
-    { 14089, 14103 }, -- Original Feet
-    { 14214, 14228 }, -- Original Legs
-    { 15265, 15267 }, -- ToAU Head
-    { 14521, 14523 }, -- ToAU Body
-    { 14928, 14930 }, -- ToAU Hand
-    { 15684, 15686 }, -- ToAU Feet
-    { 15600, 15602 }, -- ToAU Legs
-    { 16138, 16140 }, -- WotG Head
-    { 14578, 14580 }, -- WotG Body
-    { 15002, 15004 }, -- WotG Hand
-    { 15746, 15748 }, -- WotG Feet
-    { 15659, 15661 }, -- WotG Legs
+    { xi.item.FIGHTERS_MASK, xi.item.EVOKERS_HORN },           -- Original Head
```

Comment:
I get these are only ranges, but could you stylize this a bit to align beginning and end for the rows?

Reply:
Adjusted